File: 프로그래머스/unrated/181901. 배열 만들기 1/배열 만들기 1.java
import java.util.Arrays;

class Solution {
    public int[] solution(int n, int k) {
        int[] answer = new int[n];
        int count = 0;
        
        for(int i = 0; i < n; i++){
            int num = i + 1;
             if (num % k == 0) { 
                answer[count] = num;
                count++;
            }
        }
        answer = Arrays.copyOf(answer, count);
        
        return answer;
    }
}

File: 프로그래머스/unrated/181901. 배열 만들기 1/배열 만들기 1.js
function solution(n, k) {
    let answer = [];
    let count = 0;
    
    for(let i = 0; i < n; i++){
        num = i + 1;
          if (num % k == 0) { 
                answer[count] = num;
                count++;
            }
    }
    return answer;
}

